system_tests/utils.go

```diff
@@ -162,7 +162,7 @@ type Message struct {
 	MessageCount int    `json:"message_count"`
 }
 
-func rabbitmqPublishToNewQueue(rabbitmqHostName, rabbitmqPort, rabbitmqUsername, rabbitmqPassword string) error {
+func publishToNewQueue(rabbitmqHostName, rabbitmqPort, rabbitmqUsername, rabbitmqPassword string) error {
 	url := fmt.Sprintf("http://%s:%s/api/queues/%%2F/test-queue", rabbitmqHostName, rabbitmqPort)
```

Comment:
Should we extract out test-queue as it's being used at several places? Upto you as it's only in the helpers.

Reply:
I will leave it like that for now. Will address it in a different PR, I have a list of small fixes anyways. Good point!